packages/jest-cli/src/reporters/CoverageReporter.js

```diff
@@ -48,8 +48,10 @@ class CoverageReporter extends BaseReporter {
     testResult: TestResult,
     aggregatedResults: AggregatedResult,
   ) {
-    if (testResult.coverage) {
+    if (testResult.coverage && typeof testResult.coverage !== 'string') {
       this._coverageMap.merge(testResult.coverage);
+      testResult.coverage =
```

Comment:
Can we please just remove it. We'll bump a major this week anyway and I'll note this. I don't think anybody *should* be using this and I'm fine with the aggregated map being the only way to consume coverage data.